docs/sources/reference/components/otelcol/otelcol.receiver.influxdb.md

```diff
@@ -0,0 +1,164 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.influxdb/
+description: Learn about otelcol.receiver.influxdb
+title: otelcol.receiver.influxdb
+---
+
+# otelcol.receiver.influxdb
+
+`otelcol.receiver.influxdb` receives InfluxDB metrics, converts them into OpenTelemetry (OTEL) format, and forwards them to other otelcol.* components over the network.
+
+You can specify multiple `otelcol.receiver.influxdb` components by giving them different labels.
+
+## Usage
+
+```alloy
+otelcol.receiver.influxdb "influxdb_metrics" {
+  endpoint = "localhost:8086"  // InfluxDB metrics ingestion endpoint
+
+  output {
+    metrics = [...]
+  }
+}
+```
+
+## Arguments
+
+`otelcol.receiver.influxdb` supports the following arguments:
+
+| Name                     | Type           | Description                                                     | Default            | Required |
+|--------------------------|----------------|-----------------------------------------------------------------|--------------------|----------|
+| `endpoint`               | `string`       | `host:port` to listen for traffic on.                           | `"localhost:8086"` | no       |
+| `max_request_body_size`  | `string`       | Maximum request body size the server will allow.                | `20MiB`            | no       |
+| `include_metadata`       | `boolean`      | Propagate incoming connection metadata to downstream consumers. |                    | no       |
+
+By default, `otelcol.receiver.influxdb` listens for HTTP connections on `localhost`.
+To expose the HTTP server to other machines on your network, configure `endpoint` with the IP address to listen on, or `0.0.0.0:8086` to listen on all network interfaces.
+
+## Blocks
+
+The following blocks are supported inside the definition of `otelcol.receiver.influxdb`:
+
+| Hierarchy | Block          | Description                                              | Required |
+|-----------|----------------|----------------------------------------------------------|----------|
+|           | `tls`          | Configures TLS for the HTTP server.                      | no       |
+|           | `cors`         | Configures CORS for the HTTP server.                     | no       |
+|           | `debug_metrics`| Configures the metrics that this component generates.    | no       |
+|           | `output`       | Configures where to send received metrics.              | yes      |
+
+The `>` symbol indicates deeper levels of nesting. For example, `grpc > tls`
+refers to a `tls` block defined inside a `grpc` block.
```

Comment:
```suggestion
[tls]: #tls-block
[cors]: #cors-block
[debug_metrics]: #debug_metrics-block
[output]: #output-block
```

Since there's no nested block (that I see) we can drop the explanation of the hierarchy and add the anchor links to the headings.